src/core/lib/gprpp/atomic.h

```diff
@@ -28,12 +28,12 @@
 namespace grpc_core {
 
 enum class MemoryOrder {
-  RELAXED = std::memory_order_relaxed,
-  CONSUME = std::memory_order_consume,
-  ACQUIRE = std::memory_order_acquire,
-  RELEASE = std::memory_order_release,
-  ACQ_REL = std::memory_order_acq_rel,
-  SEQ_CST = std::memory_order_seq_cst
+  RELAXED = (int)std::memory_order_relaxed,
```

Comment:
nit: please use `static_cast<int>(...)` instead of c-style cast to match our style guide:
https://google.github.io/styleguide/cppguide.html#Casting

Reply:
too easy